app/code/Magento/NewRelicReporting/etc/csp_whitelist.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<csp_whitelist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Csp:etc/csp_whitelist.xsd">
+    <policies>
+        <policy id="script-src">
+            <values>
+              <value id="newrelic" type="host">newrelic.com</value>
+              <value id="newrelic-data" type="host">nr-data.com</value>
+            </values>
+        </policy>
+        <policy id="connect-src">
+            <values>
+                <value id="newrelic-data" type="host">https://bam.nr-data.net/</value>
```

Comment:
Based on [documentation](https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/install-configure/networks/#user-facing-domains) I believe that we should use wildcard `*.nr-data.net`

Reply:
I've updated with the required domains based on the documentation. I'm unsure if we need to have this domains in another section of CSP as I'm not using them on my project, but for what I understand this should cover the basics.
Let me know if there anything else to review.